packages/oml/oml.0.0.5/opam

```diff
@@ -10,8 +10,8 @@ install: [make "install"]
 remove: ["ocamlfind" "remove" "oml"]
 depends: [
   "ocamlfind" {build}
-  "lacaml"
+  "lacaml" {>= "7.2.3" & <= "7.6.2"}
```

Comment:
Updated accordingly (btw, I thought "early" oml versions meant < 0.5, not <= 0.5. My bad)